docs/astro/src/content/docs/guide/language/concepts/slint-language.mdx

```diff
@@ -4,4 +4,150 @@ title: Slint Language
 description: Slint Language
 ---
 
-Component basics. It's components all the way down...
\ No newline at end of file
+import Link from '/src/components/Link.astro';
+
+The concepts section of the guide are to give you an overview of the language from a high level. If you want to
+dive straight in then the details then the <Link type="slintFile" label="coding section"/> and language reference
+sections are the best places to start.
+
+This section is to give you an insight into the thinking behind the language and the core concepts it is made up from.
+
+
+
+## Why Slint?
+
+The Slint language helps you to describe the User Interface of your application in a declarative way.
+
+The reasons for this are that first a User Interface is quite different from abstract code. It's made up 
+of text, images, colors, animations, and so on. Even though it's a mirage, the buttons and elements do not
+really exist in the physical world, it's meant to look and behave as if it does. Buttons have pressed effects,
+lists can be flicked and behave as if they had real inertia. When being designed they are described in terms
+of UI components, forms and views. 
+
+Meanwhile the world of code is a quite different abstraction. It's made up of functions, variables, and so on.
+Even when some aspects of a UI exist such as buttons and menus they also go hand in hand with a lot of code to
+manage the implementation details.
+
+```js
+const button = document.createElement('button');
+button.textContent = 'Click me';
+document.body.appendChild(button);
+```
+
+Take this simple web example. A button is created. Then a property to show 'Click me' text is text. At this point
```

Comment:
I think we should try to avoid "easy" and "simple", as that's quite subjective. But this is definitely short :-)

```suggestion
Take this short web example. A button is created. Then a property to show 'Click me' text is text. At this point
```